uktrade/data-hub-frontend
# feature/task-list-filters

## Description of change

This change is to add several filters to the `/my-tasks` page.
This PR also includes a change to the no tasks page and makes the task tab visible regardless of if you have tasks or not.

## Test instructions

Go to `/my-tasks`. If there are tasks you should see several filters. Changing these should alter the task results accordingly.
If there are no tasks you should see something similar to the image below.

## Screenshots
With tasks:
![Screenshot 2023-12-19 at 11 52 16](https://github.com/uktrade/data-hub-frontend/assets/54268863/8ca18e1e-52e8-48fa-9d63-7369e5e63826)
Without tasks:
![Screenshot 2023-12-19 at 11 54 01](https://github.com/uktrade/data-hub-frontend/assets/14827050/1d72100c-3a86-44e6-96c8-16be56348d43)


## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
